requirements.txt

```diff
@@ -2,6 +2,7 @@ setuptools
 
 numpy>=1.9.0
 scipy>=0.14.1
+pytest
```

Comment:
I think it will be preferred not to have this here. The reason for this is that [this line](https://github.com/automl/auto-sklearn/blob/275d0d6b20d16822252d8b50bf71b1c787187f09/setup.py#L29) already provides this functionality.

This functionality can be accessed if you do something like `pip install -e .[test]` for example.